packages/react/src/components/notification/Notification.tsx

```diff
@@ -64,6 +64,11 @@ type CommonProps = React.PropsWithChildren<{
    * Note: Labels are not displayed visually for small notifications, but they are still accessible to assistive technology. This could be used to help screen reader users to better understand the context of the notification.
    */
   label?: string | React.ReactNode;
+  /**
+   * The aria label of the notification region
```

Comment:
I guess we are using two ways to write aria-label 
I think I have used both of them 😁

Reply:
Maybe we go with aria-label? I can change this:

- [ ] Change aria label -> aria-label